src/geo/jts.clj

```diff
@@ -28,24 +29,24 @@
 
 (defn get-srid
   "Gets an integer SRID for a given geometry."
-  [geom]
+  [^Geometry geom]
   (.getSRID geom))
 
 (defn set-srid
   "Sets a geometry's SRID to a new value, and returns that geometry."
-  [geom srid]
+  [^Geometry geom ^Integer srid]
   (doto geom (.setSRID srid)))
```

Comment:
hm this is a weird `doto` if we're only performing one action. We can just do `(.setSRID geom srid)`

Reply:
It was originally a two-step process of `(.setSRID geom srid)` followed by returning `geom`, since `.setSRID` doesn't actually return the modified object on its own. Whichever way you all prefer works for me!

Reply:
oh right I missed that, yeah if we're returning `geom` then `doto` is preferred.